projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -18,6 +18,59 @@ import React from 'react';
  */
 import styles from './styles.module.scss';
 
+const ConnectionItem = props => {
+	const { connections } = window.jetpackSocialInitialState;
+
+	return props.connectionIds.map( connectionId => {
+		const connection = connections[ props.provider ][ connectionId ];
+		return (
+			<div
+				key={ connectionId + Date.now().toString( 36 ) + Math.random().toString( 36 ).substr( 2 ) }
+			>
+				<tr>
+					<th className={ styles.connectionRow }> { __( 'Name', 'jetpack-social' ) }</th>
+					<th className={ styles.connectionRow }>{ __( 'Image', 'jetpack-social' ) }</th>
+				</tr>
+				<tr>
+					<td className={ styles.connectionRow }>{ connection.external_display }</td>
+					<td className={ styles.connectionRow }>
+						{ connection && (
```

Comment:
You want to check for `connection.profile_picture` here rather than in the actual `src` attribute I think.